src/data/multiset/basic.lean

```diff
@@ -1746,6 +1746,22 @@ by { simp_rw [←filter_eq', eq_comm], congr }
 
 end
 
+lemma count_true_eq_card_filter (s : multiset Prop) :
+  s.count true = (s.filter (eq true)).card :=
```

Comment:
This can be more general than just for Prop and true, it should hold for any type

Reply:
Thanks, I was confused about the type of count